src/CTA.WebForms2Blazor/ClassConverters/GlobalClassConverter.cs

```diff
@@ -171,7 +171,7 @@ private async Task<IEnumerable<UsingDirectiveSyntax>> GetAllUsingStatements()
             }
             catch (OperationCanceledException e)
             {
-                LogHelper.LogError(e, string.Format(
+                LogHelper.LogError(e, Constants.WebFormsErrorTag, string.Format(
                     Constants.CaneledServiceCallLogTemplate,
                     GetType().Name,
                     typeof(LifecycleManagerService).Name,
```

Comment:
We might need to refactor this- for this particular signature of `LogError()`, the 2nd parameter is expected to be a string template and all following params are substituted into that string template. It might be safer to prepend the tag into the log template.